core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -176,7 +176,7 @@ private[spark] class Executor(
   /**
    * When an executor is unable to send heartbeats to the driver more than `HEARTBEAT_MAX_FAILURES`
    * times, it should kill itself. The default value is 60. It means we will retry to send
-   * heartbeats about 10 minutes because the heartbeat interval is 10s.
+   * heartbeats about 10 minutes if the heartbeat interval is set to 10s or left default(10s).
```

Comment:
I think the sentence still has problems. How about: "For example, if max failures is 60 and heartbeat interval is 10s, then it will try to send heartbeats for up to 600s (10 minutes)."

Reply:
seems better. thanks ! I'will update it.

Reply:
updated. @srowen